frontend/pages/index.page.tsx

```diff
@@ -144,3 +155,89 @@ const AboutModularHistory: FC = () => {
     </Box>
   );
 };
+
+const SubscriptionBox: FC = () => {
+  const submitSubscription = () => {
+    return;
+  };
+
+  const subscriptionForm = (
+    <Grid
+      container
+      direction={"column"}
+      spacing={2}
+      alignItems={"center"}
+      justifyContent={"center"}
+      marginTop={"0rem"}
+      marginBottom={"1rem"}
+    >
+      <Grid item>
+        <TextField
+          id={"id_subscribeEmail"}
+          name={"subscribeEmail"}
+          variant={"outlined"}
+          size={"small"}
+          style={{ minWidth: "280px", marginRight: "1rem" }}
+          onKeyPress={submitSubscription}
+          label={"Email address"}
+        />
+        <Button variant="contained" sx={{ minHeight: "40px", height: "100%" }}>
+          Sign up!
+        </Button>
+      </Grid>
+    </Grid>
+  );
+
+  const subscribeDisclaimer = (
+    <Grid container>
+      <Box sx={{ fontWeight: "100" }}>
+        <Typography style={{ fontSize: "10pt" }}>
+          {"By entering your details, you are agreeing to ModularHistory's "}
+          <Link href="/privacy">
+            <a>privacy policy</a>
+          </Link>{" "}
+          and{" "}
+          <Link href="/terms">
+            <a>terms of use</a>
+          </Link>
```

Comment:
Nit: doesn't need to be changed in this PR, but I think we could use `{" and "}` instead of `{" "}and{" "}`.